src/cloudant/_common_util.py

```diff
@@ -124,19 +125,22 @@
     'group_sort': (STRTYPE, list),
     'include_docs': bool,
     'limit': (int, NONETYPE),
-    'query': (STRTYPE, int),
-    'q': (STRTYPE, int),
+    'query': (STRTYPE, int, LONGTYPE),
+    'q': (STRTYPE, int, LONGTYPE),
     'ranges': dict,
     'sort': (STRTYPE, list),
     'stale': STRTYPE,
     'highlight_fields': list,
     'highlight_pre_tag': STRTYPE,
     'highlight_post_tag': STRTYPE,
-    'highlight_number': (int, NONETYPE),
-    'highlight_size': (int, NONETYPE),
+    'highlight_number': (int, LONGTYPE, NONETYPE),
+    'highlight_size': (int, LONGTYPE, NONETYPE),
     'include_fields': list
 }
 
+# Test long type in Python 2
+LONG_NUMBER = 92233720368547758070
```

Comment:
I tried that before but `SyntaxError: invalid syntax` will always be thrown if `L` syntax exists when running tests in Python 3.